domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputIsEquivalentToRuleClassifierProviderTest.kt

```diff
@@ -156,7 +175,7 @@ class FractionInputIsEquivalentToRuleClassifierProviderTest {
   }
 
   @Test
-  fun testEquality_fraction2Over33Answer_withFaction1Over242Input_bothValuesNotEquivalent() {
+  fun testEquality_fraction33Over2Answer_withFraction242Over1Input_bothValuesNotEquivalent() {
```

Comment:
This & the test above seem like repeats of the earlier same-fractions-are-equivalent, different-fractions-not-equivalent, fractions-that-are-multiples-of-each-other-are-equivalent. Do these tests add anything?

Reply:
I agree -- I think the nuance is it being an improper fraction but not sure if that's a unique case at all.
Removed these two